### ISSUE TITLE
Comment placement in render causing istanbul issues

### ISSUE THREAD

**Issue by wavded** [babel/babel#1583]
Given:

```js
import cfg from '_/config'
import log from '_/logger'
import mongoose from 'mongoose'

/* istanbul ignore next */
export function connect () {
  mongoose.connect(cfg.mongodb.uri)
  mongoose.connection.on('error', log.crit)
}
```

Produces:

```js
'use strict';

Object.defineProperty(exports, '__esModule', {
  value: true
});

/* istanbul ignore next */
exports.connect = connect;

function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { 'default': obj }; }

var _config = require('_/config');

var _config2 = _interopRequireDefault(_config);

var _logger = require('_/logger');

var _logger2 = _interopRequireDefault(_logger);

var _mongoose = require('mongoose');

var _mongoose2 = _interopRequireDefault(_mongoose);

function connect() {
  _mongoose2['default'].connect(_config2['default'].mongodb.uri);
  _mongoose2['default'].connection.on('error', _logger2['default'].crit);
}
```

Which istanbul will no longer ignore since the comment isn't by the function definition anymore.  Is it possible to get comments next to the function definition whilst exporting?

FWIW, this is workaround:

```js
import cfg from '_/config'
import log from '_/logger'
import mongoose from 'mongoose'

/* istanbul ignore next */
function connect () {
  mongoose.connect(cfg.mongodb.uri)
  mongoose.connection.on('error', log.crit)
}

export { connect as connect }
```

**Comment by sebmck** [comment]
Fixed as of 5.4.5, thanks!

**Comment by wavded** [comment]
thx @sebmck !
